sycl/include/sycl/sub_group.hpp

```diff
@@ -42,6 +42,23 @@ namespace sub_group {
 template <typename T>
 using SelectBlockT = select_cl_scalar_integral_unsigned_t<T>;
 
+template <typename MultiPtrTy> auto convertToBlockPtr(MultiPtrTy MultiPtr) {
+  static_assert(is_multi_ptr_v<MultiPtrTy>);
+  auto DecoratedPtr = convertToOpenCLType(MultiPtr);
+  using DecoratedPtrTy = decltype(DecoratedPtr);
+  using ElemTy = remove_decoration_t<std::remove_pointer_t<DecoratedPtrTy>>;
+
+  using TargetElemTy = SelectBlockT<ElemTy>;
```

Comment:
If ElemTy has cv qualifier, it looks like SelectBlockT is going to drop that qualifier.
Maybe my understanding is wrong.

Reply:
True, but I don't think it's a regression caused by this PR.

Reply:
I see. 
NIT: In that case it will be nice to have some todo maybe, otherwise I guess it will be forgotten.